aes-siv/src/siv.rs

```diff
@@ -263,11 +268,11 @@ where
 /// initialization vector (for AES-CTR encryption) and MAC.
 fn s2v<M, I, T>(mac: &mut M, headers: I, message: &[u8]) -> Result<Tag, Error>
 where
-    M: Mac<OutputSize = U16>,
+    M: Mac<OutputSize = U16> + FixedOutputReset,
     I: IntoIterator<Item = T>,
     T: AsRef<[u8]>,
 {
-    mac.update(&Tag::default());
+    Mac::update(mac, &Tag::default());
```

Comment:
I'm still not exactly sure why but I had to disambiguate these /cc @newpavlov 

```
   Compiling aes-siv v0.7.0-pre (/Users/tony/src/RustCrypto/AEADs/aes-siv)
error[E0034]: multiple applicable items in scope
   --> aes-siv/src/siv.rs:275:9
    |
275 |     mac.update(&Tag::default());
    |         ^^^^^^ multiple `update` found
    |
note: candidate #1 is defined in the trait `Mac`
   --> /Users/tony/.cargo/registry/src/github.com-1ecc6299db9ec823/digest-0.10.3/src/mac.rs:38:5
    |
38  |     fn update(&mut self, data: &[u8]);
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: candidate #2 is defined in the trait `Update`
   --> /Users/tony/.cargo/registry/src/github.com-1ecc6299db9ec823/digest-0.10.3/src/lib.rs:77:5
    |
77  |     fn update(&mut self, data: &[u8]);
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: disambiguate the associated function for candidate #1
    |
275 |     Mac::update(&mut mac, &Tag::default());
    |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
help: disambiguate the associated function for candidate #2
    |
275 |     Update::update(&mut mac, &Tag::default());
    |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

...which occurs even if the `Update` trait hasn't been imported or bounded upon, nor is it a bound of `Mac`, so I'm not sure why this is ambiguous